### PR TITLE
honcho: 0.5.0

### PR DESCRIPTION
**Pull request by robsonpeixoto**

### CODE REVIEW

File: Library/Formula/honcho.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  url "https://pypi.python.org/packages/source/h/honcho/honcho-0.5.0.tar.gz"
+  sha1 "c9f9bdaa59293f8c9ac515d55c9ee4a09240821d"
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
```
Comment (bfontaine) [inline Library/Formula/honcho.rb]: Could you add a test? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (mikemcquaid) [inline Library/Formula/honcho.rb]: :+1:

Comment (robsonpeixoto) [inline Library/Formula/honcho.rb]: Done in 5f37dedff65f3d4335bbc697c277abe8abdfbf16

**Comment by robsonpeixoto** [comment]
In my machine the test are working. Can you(@bfontaine) check it, please?

File: Library/Formula/honcho.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  test do
+    (testpath/'Procfile').write <<-EOS.undent
+      talk: echo $MY_VAR
+    EOS
+    (testpath/'.env').write <<-EOS.undent
+      MY_VAR=hi
+    EOS
+
```
Comment (bfontaine) [inline Library/Formula/honcho.rb]: There are trailing spaces here.

File: Library/Formula/honcho.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+  test do
+    (testpath/'Procfile').write <<-EOS.undent
```
Comment (bfontaine) [inline Library/Formula/honcho.rb]: Prefer double quotes (`"`) instead of single ones (`'`).

File: Library/Formula/honcho.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      talk: echo $MY_VAR
+    EOS
+    (testpath/'.env').write <<-EOS.undent
```
Comment (bfontaine) [inline Library/Formula/honcho.rb]: See my comment above about quotes.

**Comment by bfontaine** [comment]
@robsonpeixoto The test is working :+1: the build [fails](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20688/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_honcho___strict/) because of the `audit` part, you can check locally with:

    brew audit honcho --strict

I added some comments on the code to match the audit warnings. Feel free to ask any question.

**Comment by robsonpeixoto** [comment]
```bash
$ brew update
Already up-to-date.
$ brew audit honcho --strict
==> brew style honcho

1 file inspected, no offenses detected
```

**Comment by bfontaine** [comment]
Great! Now the build [failed on ML](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20706/version=mountain_lion/console) but that’s a git error, poke @mikemcquaid:

```
==> brew pull --clean https://github.com/Homebrew/homebrew/pull/36375	 FAILED
fatal: ref HEAD is not a symbolic ref
==> Applying patch
Applying: honcho: 0.5.0
Applying: honcho: add test
Applying: honcho: code style
Auto packing the repository for optimum performance. You may also
run "git gc" manually. See "git help gc" for more information.
fatal: Unable to create '/usr/local/.git/refs/remotes/origin/pr/24653/head.lock': File exists.

If no other git process is currently running, this probably means a
git process crashed in this repository earlier. Make sure no other git
process is running and remove the file manually to continue.
error: failed to run reflog
Resolve operation not in progress, we are not resuming.
Error: Patch failed to apply: aborted.
```

**Comment by tdsmith** [comment]
Thanks @robsonpeixoto, @bfontaine! Added in 8641c0f517e; forgot to mention the issue when I squashed, oops.

**Pull request closed without merging**